tests/js/realm-tests.js

```diff
@@ -397,18 +415,28 @@ module.exports = BaseTest.extend({
             realm.create('PersonObject', {name: 'Alex', age: 12, married: true});
         });
 
-        TestCase.assertThrows(function() { 
+        // Should be able to pass constructor for getting objects.
+        var objects = realm.objects(schemas.PersonObject);
```

Comment:
Maybe move this down to the part of the testcase where this data is actually used.